app/src/background.js

```diff
@@ -352,40 +329,40 @@ async function createOrUpdateUntranslatedFakeNode(
       return;
     }
 
-    const oldTitle = translatedElement?.textContent ?? fakeNode?.textContent;
+    const oldTitle =
```

Comment:
Maybe simplify it? Looks too hard to read (change ternary into if expression with comment)